### PR TITLE
Misc small refactor cleanup for OIDC Jakarta work

### PR DESCRIPTION
**Pull request by ayoho**
Moves some unit tests from a class that was refactored into their new, appropriate unit test classes.

### CODE REVIEW

**Comment by ayoho** [comment]
#build

File: dev/com.ibm.ws.security.common/test/com/ibm/ws/security/common/crypto/HashUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
```
Comment (jimmy1wu) [inline dev/com.ibm.ws.security.common/test/com/ibm/ws/security/common/crypto/HashUtilsTest.java]: 2020 -> 2022

**Comment by ayoho** [comment]
#libby

**Comment by ayoho** [comment]
#libby

**Review by jimmy1wu — APPROVED** [review]

**Pull request merged**